src/olympia/hero/tests/test_views.py

```diff
@@ -63,6 +78,7 @@ def test_basic(self):
             # - 1 to fetch the add-ons authors
             # - 1 to fetch the add-ons version previews (for static themes)
             # - 1 to fetch the add-ons previews
+            # - 3x: 1 to fetch the primary hero image
```

Comment:
See my comment - we should be able to get this back to 11 by using `select_related`